scene/3d/physics/rigid_body_3d.cpp

```diff
@@ -658,16 +658,18 @@ void RigidBody3D::_reload_physics_characteristics() {
 	}
 }
 
-PackedStringArray RigidBody3D::get_configuration_warnings() const {
-	PackedStringArray warnings = CollisionObject3D::get_configuration_warnings();
+#ifdef TOOLS_ENABLED
+LocalVector<ConfigurationInfo> RigidBody3D::get_configuration_info() const {
+	LocalVector<ConfigurationInfo> infos = CollisionObject3D::get_configuration_info();
```

Comment:
```suggestion
	LocalVector<ConfigurationInfo> infos = PhysicsBody3D::get_configuration_info();
```
Use the direct parent (thought I had caught this one in my recent PR but must have missed it)